content/CORE/System/Update/_index.md

```diff
@@ -123,7 +123,7 @@ Be aware of these caveats before attempting a major version upgrade:
   Unless new ZFS feature flags are needed, it is safe to leave the pool at the current version.
   If the pool is upgraded, it will not be possible to boot into a previous TrueNAS version that does not support the newer feature flags.
 * Upgrading the firmware of Broadcom SAS HBAs to the latest version is recommended.
```

Comment:
Not sure how best to fix this list's parallel construction issues other than moving this point to the end and not have it part of the bullet list or create a second bullet list with line 127 which would work.

Reply:
I say we just remove the bullets and use the bold segments as quasi-headers for each point.